src/axom/primal/geometry/NumericArray.hpp

```diff
@@ -479,7 +474,10 @@ template <typename T, int SIZE>
 void NumericArray<T, SIZE>::to_array(T* arr) const
 {
   SLIC_ASSERT(arr != nullptr);
-  memcpy(arr, m_components, NBYTES);
```

Comment:
Was the (old) call to `memcpy` the only place that used `NBYTES`?

Reply:
@agcapps Yes, this is it.